src/controllers/tasks/people.with.significant.control.controller.ts

```diff
@@ -54,17 +64,32 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
     const pscButtonValue = req.body.pscRadioValue;
 
     if (!pscButtonValue) {
-      const psc: PersonOfSignificantControl = await getPscData(req) as PersonOfSignificantControl;
-      const pscAppointmentType = psc.appointmentType;
-      const formattedPsc: PersonOfSignificantControl = formatPSCForDisplay(psc);
-      const pscTemplateType: string = getPscTypeTemplate(pscAppointmentType);
+      const pscs: PersonOfSignificantControl[] | undefined = await getPscData(req);
+      const pscList = new Array(0);
+      if (!pscs) {
+        throw createAndLogError(`No PSC data found, no radio button selected`);
+      }
+      for (const psc of pscs) {
```

Comment:
is this whole loop the same as above, could it be moved out into a local controller method below?